### DISCUSSION TITLE
Editing the app in Android Studio

### DISCUSSION THREAD

**Discussion by meirtorj** [AnySoftKeyboard/AnySoftKeyboard#3012] [Q&A]
I try to build and get this error:
Caused by: org.gradle.api.internal.tasks.DefaultTaskContainer $ TaskCreationException: Could not create task ': ime: app: minifyCanaryWithR8'.
what can we do?

**Comment by menny** [comment]
does this happen in command-line or inside Android Studio?
Which version of Java are you using?

**Reply by meirtorj** [reply]
Inside Android Studio. 
Java 11

**Reply by meirtorj** [reply]
Can I have help please?
